python/tests/voice/test_elevenlabs_turn_commit.py

```
@@ -240,43 +240,78 @@ async def test_committed_user_message_is_server_accepted_shape():
+        await adapter._pump_tick()
+    emitted = [base64.b64decode(c) for c in socket.audio_chunks]
+    assert all(len(f) == 960 for f in emitted)
+    assert sum(len(f) for f in emitted if f == b"\x00" * 960) >= 16000
 
 
 @pytest.mark.asyncio
 async def test_silence_tail_bytes_resizes_the_fallback_tail():
-    """``silence_tail_bytes`` resizes the legacy fallback tail."""
+    """``silence_tail_bytes`` resizes the fallback tail — the number of queued
+    960-byte silence frames scales with it."""
     adapter, socket = await _connected_adapter(
```

Ruff (0.15.20)

[warning] Unpacked variable `socket` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)